src/inodes/INodeManager.ts

```diff
@@ -743,14 +743,14 @@ class INodeManager {
     for await (const data of dataDb.createReadStream(options)) {
       // This is to account for the case where a some blocks are missing in a database
       // i.e. blocks 0 -> 3 have data and a write operation was performed on blocks 7 -> 8
-      while (inodesUtils.unbufferId((data as any).key) !== blockCount) {
+      while (inodesUtils.unbufferId((data as any).key) != blockCount) {
```

Comment:
Why did you change this? The `!==` should be used for anything that is not a null comparison.

Reply:
I copied some of the code over so that must have been in the old code, Ill fix that now